src/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Changelog
 
 ## [Unreleased]
-- Added Unity.Mathematics.Extras.Plane to represent planes in 3D space.
+- Added Unity.Mathematics.Geometry.Plane to represent planes in 3D space.
 - Adding Random.CreateFromIndex() to assist in creating Random instances from loop indices.
+- Added more MinMaxAABB functionality from Unity.Physics.Aabb.
```

Comment:
Is this a useful changelog entry when the MinMaxAABB is not public?

Reply:
It's not useful and I haven't found a good workflow for this but I think it's better than the alternative which is for me to have to dig through all the commits before a release to come up with the changelog. The way I'm thinking of how this works is I add things to the changelog when I make the commits and then before release I go back to review the unreleased items and make the appropriate changes to code to make sure it's public and update the changelog as needed.